### PR TITLE
Make spellcheck for pkg/* (TODO:*_test.go)

### PR DESCRIPTION
**Pull request by ghost**
**What this PR does / why we need it**: Spellcheck for readability

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**: more spellcheck to do

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

**Comment by aaron12134** [comment]
@dims Could you lift a finger to put a LGTM label to this PR now that the CI test is passed?

**Comment by dims** [comment]
@aaron12134 apologies. i had temporary privileges for 1.5 release cycle only. I don't have LGTM privs anymore.

**Comment by xulike666** [comment]
@dims , thank you anyway.  @erictune , could you lift a finger to put a LGTM label to this PR now that the CI test is passed?

File: pkg/apiserver/proxy_test.go [hunk 1]
```diff
@@ -127,7 +127,7 @@ func TestProxyRequestContentLengthAndTransferEncoding(t *testing.T) {
 				"Content-Encoding":  nil, // none set
 				"Transfer-Encoding": nil, // Transfer-Encoding gets removed
 			},
-			expectedBody: sampleData, // sample data is unchunked
+			expectedBody: sampleData, // sample data is unchecked
```
Comment (lavalamp) [inline pkg/apiserver/proxy_test.go]: revert, unchunked is legit

File: pkg/apiserver/proxy_test.go [hunk 2]
```diff
@@ -142,7 +142,7 @@ func TestProxyRequestContentLengthAndTransferEncoding(t *testing.T) {
 				"Content-Encoding":  []string{"gzip"},
 				"Transfer-Encoding": nil, // gets removed
 			},
-			expectedBody: zip(sampleData), // sample data is unchunked, but content-encoding is preserved
+			expectedBody: zip(sampleData), // sample data is unchecked, but content-encoding is preserved
```
Comment (lavalamp) [inline pkg/apiserver/proxy_test.go]: revert

**Comment by lavalamp** [comment]
Thank you for fixing our code. I actually recommend splitting changes like this into multiple small PRs, that way you won't have to rebase frequently. All of these changes are good except that "unchunked" is actually talking about whether a http transfer is chunked or not--don't change it.

**Pull request closed without merging**